packages/nuekit/src/layout.js

```diff
@@ -53,7 +54,7 @@ export function renderHead(data, is_prod) {
   if (components) pushMeta('nue:components', components.map(uri => `${base}${uri}`).join(' '))
 
   // misc
-  if (favicon) head.push(`<link rel="shortcut icon" src="${favicon}">`)
+  if (favicon) head.push(`<link rel="shortcut icon" type="${TYPES[extname(favicon).slice(1)]}" src="${favicon}">`)
```

Comment:
maybe should to lower case here? otherwise, .PNG doesn't work

Reply:
Nicely spotted TYPES in nueserver.js! I think it's good to merge like this for now. I think we should add a getMime() method to nueserver later.

Reply:
> maybe should to lower case here? otherwise, .PNG doesn't work

Agreed!

> Nicely spotted TYPES in nueserver.js!

:wink: 

> getMime() method to nueserver

Sounds good

Not sure, but it would be probably good to expand the list with [more common types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types). 
It was quite interesting looking at a PDF in text format on the browser. Even though it is just the dev server.

Reply:
Maybe even a package, which has all the up-to-date types like [`mrmime`](https://github.com/lukeed/mrmime) or [`mime`](https://github.com/broofa/mime)?